sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/PemReaderTests.cs

```diff
@@ -89,7 +89,7 @@ public void LoadCertificatePemOverridesCer()
         [Test]
         public void LoadRSACertificate()
         {
-#if NET461 || NETCOREAPP2_1
+#if NET461 || NETCOREAPP
```

Comment:
I think you'd just want to get rid of the RHS:
```suggestion
#if NET461
```